### ISSUE TITLE
[Feature] Native support Azure pipeline+ Playwright + run in parallel on multiple test agents

### ISSUE THREAD

**Issue by python012** [microsoft/playwright#27014]
Hi, I did some google research and it looks like now only VSTest is supported to run on multiple test agents, may I know is there a native solution or best practice for Playwright tests?

Now I have to do it using `--grep @part1` and `--grep @part2`, and try to launch the 2 part of tests in 2 different Azure pipeline stages(with different test agent)

**Comment by dgozman** [comment]
@python012 I'd recommend to look into [parallelism](https://playwright.dev/docs/test-parallel) and [sharding](https://playwright.dev/docs/test-sharding) guides. If that's not enough, please describe your usecase in details - right now I cannot really understand the issue and help you.

**Comment by dgozman** [comment]
Closing as per above. If you still encounter problems, please file a new issue with a repro and link to this one.
